apps/modernization-ui/src/apps/patient/profile/sexBirth/BirthAndGenderEntryFields.tsx

```diff
@@ -13,20 +13,22 @@ import { usePatientSexBirthCodedValues } from './usePatientSexBirthCodedValues';
 const UNKNOWN_GENDER = 'U';
 
 export const BirthAndGenderEntryFields = () => {
-    const { control } = useFormContext<BirthAndGenderEntry>();
+    const { control, setValue } = useFormContext<BirthAndGenderEntry>();
     const currentBirthday = useWatch({ control, name: 'birth.bornOn' });
-
     const age = useMemo(() => calculateAge(currentBirthday), [currentBirthday]);
-
     const selectedCurrentGender = useWatch({ control, name: 'gender.current' });
-
     const selectedState = useWatch({ control, name: 'birth.state' });
-
     const selectedMultipleBirth = useWatch({ control, name: 'birth.multipleBirth' });
 
     const coded = usePatientSexBirthCodedValues();
 
-    const byState: CountiesCodedValues = selectedState ? useCountyCodedValues(selectedState) : { counties: [] };
+    const { counties } = useCountyCodedValues(selectedState ?? '');
```

Comment:
question (non-blocking): For the list, would something like this [work](https://github.com/CDCgov/NEDSS-Modernization/blob/7836fc0fd5393637c9d50584d902543ddc40a287/apps/modernization-ui/src/apps/patient/profile/addresses/AddressEntryFields.tsx#L14)?